library/src/morpheus-master/js/src/sakai.morpheus.usernav.js

```diff
@@ -1,10 +1,9 @@
-/**
- * ESC handler to dismiss user nav
- */
+let modalVisible = false;
 
+// ESC handler to dismiss user nav
 function userNavEscHandler(e){
 
-  if (e.keyCode === 27) { // esc keycode
+  if (e.keyCode === 27 && modalVisible != true) { // esc keycode
```

Comment:
!modalVisible. I'm consufed about this, should be modalVisible?

Reply:
It was a quick fix. `!modalVisible` condition prevents the profile menu from closing when the Editor modal is active and you press ESC.

This way, once you close the Editor modal, the focus will be able to get back to the initial element, in this case, the avatar img on the profile menu.

![focusBehaviourCrop](https://user-images.githubusercontent.com/81161239/153871548-dd6c8cd6-74e1-4e3c-8a1d-8f9bec8f29d0.png)

A better solution might involve remaking the profile menu, perhaps using the BS modal classes.